src/screening/ScreeningFilters.jsx

```diff
@@ -1,50 +1,103 @@
 import React from "react";
 import {
   Dropdown,
-  FieldInput
 } from '@openmrs/react-components';
-import {
-  Glyphicon,
-  FormControl,
-} from 'react-bootstrap';
+import { FormControl } from 'react-bootstrap';
+import utils from "../utils";
 import './screeningFilters.css';
 
-let ScreeningFilters = props => (
```

Comment:
Let's rename this component to "IdentifierFilter"

Also, we can get rid of the complexity of determining the prefix by loading the row data... we can just hard the prefixes...